CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### 2.0.1 (Next)
 
 * Your contribution here.
+* [#434](https://github.com/slack-ruby/slack-ruby-client/pull/434): Fix incompatibility with Faraday >= 2.7.0 and GLI usage in the `slack` script [@blowfishpro](https://github.com/blowfishpro)
```

Comment:
This needs to match the format of other entries, it's missing a `- before your github account, and a period at the end.